TESTS/mbedtls/multi/main.cpp

```diff
@@ -163,5 +174,18 @@ utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
 Specification specification(greentea_test_setup, cases, greentea_test_teardown_handler);
 
 int main() {
-    Harness::run(specification);
+    int ret = 0;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+    if((ret = mbedtls_platform_setup(&platform_ctx))!= 0)
+    {
+        mbedtls_printf("Mbed TLS selftest failed! mbedtls_platform_setup returned %d\n", ret);
```

Comment:
Log message is incorrect.

Reply:
@mazimkhan What would you like to be different here?

Reply:
I believe that Azim is talking about the "selftest" log in a "multi" test. Copy and paste error on my side, I'll fix that.

Reply:
Yes and it could say "platform setup failed".